.travis.yml

```diff
@@ -7,25 +7,21 @@ node_js:
   - "6"
   - "8"
 
-before_install:
-  # Minimal installation as desdcribed in https://github.com/martinda/gnu-parallel
-  - wget http://git.savannah.gnu.org/cgit/parallel.git/plain/src/parallel
-  - chmod +x parallel
-  - cp parallel sem
-  - sudo mv parallel sem /usr/local/bin
+env:
+  global:
+    secure: I9sReagJiE7B0154AnU3t8WcdPtmZm86vMp+6umKegvzYr46jbypPAmlWW50A09fw4AmTaGOtwpYtvglf0JikkR6saXY0rRquDnO70DRmbdrO2o2WMjjCzkYID58SkGvqsqVJ4HXcu0HdJp6o5L7v7JMdzji6abYKxtsnw2ouk0=
+
+  matrix:
+    - TEST_SCRIPT=build
+    - TEST_SCRIPT=test
+    - TEST_SCRIPT=fix
```

Comment:
Perhaps, update is a better name -- now that we rely on Travis to update example (and optionally schema)?   (Though the prettifier part is technically still a fix.)